packages/extensions/openapi-to-typespec/src/utils/discriminator.ts

```diff
@@ -10,7 +10,11 @@ export function getDiscriminator(schema: ObjectSchema): TypespecObjectProperty |
   if (!schema.discriminatorValue) {
     return undefined;
   }
-  const { serializedName: name, language } = getDiscriminatorProperty(schema);
+  const discriminator = getDiscriminatorProperty(schema);
+  if (!discriminator) {
+    throw new Error("No discriminator property found");
```

Comment:
Is this a severe error that we need to stop the converter?  If not, we could just output some log and continue, and later manually fix those problems.